src/SFA.DAS.RoatpGateway.Web.UnitTests/Controllers/ExperienceAndAccreditation/SubcontractorDeclarationTests.cs

```diff
@@ -78,108 +86,112 @@ public async Task saving_subcontractor_declaration_saves_evaluation_result()
-                SourcesCheckedOn = DateTime.Now,
-                ErrorMessages = new List<ValidationErrorDetail>(),
-                ApplicationId = applicationId,
-                PageId = pageId
-            };
-
-            var command = new SubmitGatewayPageAnswerCommand(vm);
-
-            GatewayValidator.Setup(v => v.Validate(command))
-                .ReturnsAsync(new ValidationResponse
-                {
-                    Errors = new List<ValidationErrorDetail>
-                        {
-                            new ValidationErrorDetail {Field = "OptionFail", ErrorMessage = "needs text"}
-                        }
-                }
-                );
-
-            _orchestrator.Setup(x => x.GetSubcontractorDeclarationViewModel(It.Is<GetSubcontractorDeclarationRequest>(y => y.ApplicationId == vm.ApplicationId
-                                                                                && y.UserName == Username))).ReturnsAsync(vm);
-
-            await _controller.EvaluateSubcontractorDeclarationPage(command);
-
-            ApplyApiClient.Verify(x => x.SubmitGatewayPageAnswer(It.IsAny<Guid>(), It.IsAny<string>(), It.IsAny<string>(), It.IsAny<string>(), It.IsAny<string>(), It.IsAny<string>(), It.IsAny<string>()), Times.Never);
-        }
-
-        [Test]
-        public async Task clarifying_subcontractor_declaration_without_required_fields_does_not_save()
-        {
-            var applicationId = Guid.NewGuid();
-            var pageId = GatewayPageIds.SubcontractorDeclaration;
-
-            var vm = new SubcontractorDeclarationViewModel
-            {
-                Status = SectionReviewStatus.Fail,
-                SourcesCheckedOn = DateTime.Now,
-                ErrorMessages = new List<ValidationErrorDetail>(),
-                ApplicationId = applicationId,
-                PageId = pageId,
-                ClarificationAnswer = ClarificationAnswer
-            };
-
-            var command = new SubmitGatewayPageAnswerCommand(vm);
-
-            GatewayValidator.Setup(v => v.ValidateClarification(command))
-                .ReturnsAsync(new ValidationResponse
-                    {
-                        Errors = new List<ValidationErrorDetail>
-                        {
-                            new ValidationErrorDetail {Field = "OptionFail", ErrorMessage = "needs text"}
-                        }
-                    }
-                );
-
-            _orchestrator.Setup(x => x.GetSubcontractorDeclarationViewModel(It.Is<GetSubcontractorDeclarationRequest>(y => y.ApplicationId == vm.ApplicationId
-                && y.UserName == Username))).ReturnsAsync(vm);
-
-            await _controller.ClarifySubcontractorDeclarationPage(command);
-
-            ApplyApiClient.Verify(x => x.SubmitGatewayPageAnswer(It.IsAny<Guid>(), It.IsAny<string>(), It.IsAny<string>(), It.IsAny<string>(), It.IsAny<string>(), It.IsAny<string>(), It.IsAny<string>()), Times.Never);
-        }
+        //[Test]
```

Comment:
As discussed, there is a way to sent in files. Assuming these are to be picked up when you get back in January